i18n/bundle/bundle.go

```diff
@@ -134,6 +134,25 @@ func (b *Bundle) Translations() map[string]map[string]translation.Translation {
 	return b.translations
 }
 
+// LanguageTags returns the tags of all languages that that have been added.
+func (b *Bundle) LanguageTags() (tags []string) {
```

Comment:
Sorry, I didn't realize you had already made some comments (I kind of hate how comments disappear after a new diff is uploaded).

In any case, I think it is best to avoid named return values except for a few circumstances.
  1. Your function returns multiple values of the same type so you need a name to disambiguate.
  2. Your function uses defer to modify return values before actually returning.

Even when I have named return values due to above exceptions, I would always still use explicit return statements because it is easier to read.

In this particular case, the named return value just unnecessarily stutters the method signature.